slowfast/datasets/ava_dataset.py

```diff
@@ -75,7 +75,7 @@ def _load_data(self, cfg):
         (
             self._keyframe_indices,
             self._keyframe_boxes_and_labels,
-        ) = ava_helper.get_keyframe_data(boxes_and_labels)
+        ) = ava_helper.get_keyframe_data(self.cfg, boxes_and_labels)
```

Comment:
You could've used just `cfg`, as you get it as an input. It changes nothing, so just as a note.

Reply:
I suggest you indeed use `cfg` to avoid coherence problems.